hack/create-installation.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+SOURCE_PATH="$(dirname $0)/.."
+TMP_DIR="$(mktemp -d)"
+INSTALLATION_PATH="${TMP_DIR}/installation.yaml"
+
+REGISTRY="$(${SOURCE_PATH}/hack/get-cd-registry.sh)"
+
+cat << EOF > ${INSTALLATION_PATH}
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Installation
+metadata:
+  name: provider-aws
+spec:
+  componentDescriptor:
+    ref:
+      repositoryContext:
+        type: ociRegistry
+        baseUrl: ${REGISTRY}
+      componentName: github.com/gardener/gardener-extension-provider-aws
+      version: ${EFFECTIVE_VERSION}
+
+  blueprint:
+    ref:
+      resourceName: blueprint
+
+  imports:
+    targets:
+      - name: cluster
+        target: "#cluster"
+
+  importDataMappings:
+    cloudProfile:
+      machineImages:
+        - name: gardenlinux
+          versions:
+          - classification: preview
+            cri:
+            - containerRuntimes:
+              - type: gvisor
+              name: containerd
+            version: 318.4.0
+          - classification: supported
+            cri:
+            - containerRuntimes:
+              - type: gvisor
+              name: containerd
+            version: 184.0.0
+        - name: flatcar
+          versions:
+          - classification: preview
+            version: 2605.11.0
+
+      regions:
+        - name: ap-northeast-1
+          zones:
+          - name: ap-northeast-1a
+          - name: ap-northeast-1c
+          - name: ap-northeast-1d
+        - name: ap-northeast-2
+          zones:
+          - name: ap-northeast-2a
+          - name: ap-northeast-2b
+          - name: ap-northeast-2c
+          - name: ap-northeast-2d
+
+    kubernetesVersions:
+      - classification: supported
+        version: 1.20.6
+      - classification: deprecated
+        expirationDate: '2021-09-15T23:59:59Z'
+        version: 1.20.5
+      - classification: deprecated
+        expirationDate: '2021-08-15T23:59:59Z'
+        version: 1.20.4
+      - classification: deprecated
+        expirationDate: '2021-07-31T23:59:59Z'
+        version: 1.20.2
```

Comment:
Same as above, those values will need to be actively maintained and will just introduce unwanted effort.

Reply:
I have removed that